### ISSUE TITLE
Support multi-line imports

### ISSUE THREAD

**Issue by fleischie** [fleischie/vim-styled-components#17]
Having the following import statement does not trigger the correct filetype detection:
```javascript
import styled, {
  css,
  keyframes,
} from 'styled-components';
```

Maybe the regex can be extended to include multi-line import statements.

**Comment by fleischie** [comment]
This should be included in the next release, which will include this and #18 (hopefully soonish).
